xl_auth/permission/forms.py

```diff
@@ -60,6 +60,10 @@ def set_defaults(self, user_id, collection_id):
         """Apply 'user_id' and 'collection_id' field defaults."""
         self.user_id.default = user_id
         self.collection_id.default = collection_id
+        if not self.current_user.is_admin:
+            self.cataloging_admin.render_kw = {'disabled': 'disabled',
```

Comment:
I tried it out, and it works. I thought it was a good idea, but when actually trying it out on my computer I'd say it's even more obtuse than before. At least you got a clear message. Annoying perhaps, but plain and instructive. Finding that spot to hover for help text to appear, along with very weak visual signals, makes it kinda horrible...

I'd like to propose simply hiding it (visually) in registration view template, e.g.
```jinja
<label class="checkbox" style="display: {{ 'initial' if current_user.is_admin else 'none' }}">
```

I.e.

<img width="506" alt="screen shot 2017-12-28 at 11 01 02 am" src="https://user-images.githubusercontent.com/786326/34407565-4654432e-ebbf-11e7-9226-56e512cb683c.png">


And keep the edit form as it is. Or make it much clearer what they can and cannot do.